src/centralstore/incremental/RelationBlock.cpp

```diff
@@ -0,0 +1,332 @@
+            "relation block address " +
+            std::to_string(address));
+        return NULL;
+    }
+
+    if (!RelationBlock::relationsDB->read(reinterpret_cast<char*>(&destination.nextPid),
+                                          RECORD_SIZE)) {  // < ------ relation data offset ID = 7
+        relation_block_logger.error(
+            "ERROR: Error while reading relation destination next relation partition id offset ID = 7 from "
+            "relation block address " +
+            std::to_string(address));
+        return NULL;
+    }
+
+    if (!RelationBlock::relationsDB->read(reinterpret_cast<char*>(&destination.preRelationId),
+                                          RECORD_SIZE)) {  // < ------ relation data offset ID = 8
+        relation_block_logger.error(
+            "ERROR: Error while reading relation destination previous relation address data offset ID = 8 from "
+            "relation block address " +
+            std::to_string(address));
+        return NULL;
+    }
+
+    if (!RelationBlock::relationsDB->read(reinterpret_cast<char*>(&destination.prePid),
+                                          RECORD_SIZE)) {  // < ------ relation data offset ID = 9
+        relation_block_logger.error(
+            "ERROR: Error while reading relation destination previous relation partition id data offset ID = 9 from "
+            "relation block address " +
+            std::to_string(address));
+        return NULL;
+    }
+
+    if (!RelationBlock::relationsDB->read(reinterpret_cast<char*>(&propertyReference),
+                                          RECORD_SIZE)) {  // < ------ relation data offset ID = 10
+        relation_block_logger.error(
+            "ERROR: Error while reading relation property address data offset ID = 10 from relation block address " +
+            std::to_string(address));
+        return NULL;
+    }
+
+    return new RelationBlock(address, source, destination, propertyReference);
+}
+
+RelationBlock* RelationBlock::nextSource() { return RelationBlock::get(this->source.nextRelationId); }
+
+RelationBlock* RelationBlock::previousSource() { return RelationBlock::get(this->source.preRelationId); }
+
+RelationBlock* RelationBlock::nextDestination() { return RelationBlock::get(this->destination.nextRelationId); }
+
+RelationBlock* RelationBlock::previousDestination() { return RelationBlock::get(this->destination.preRelationId); }
+
+bool RelationBlock::setNextSource(unsigned int newAddress) {
+    if (this->updateRelationRecords(2, newAddress)) {
+        this->source.nextRelationId = newAddress;
+    } else {
+        throw "Exception: Error while updating the relation next source address " + std::to_string(newAddress);
+    }
+    return true;
+}
+
+bool RelationBlock::setPreviousSource(unsigned int newAddress) {
+    if (this->updateRelationRecords(4, newAddress)) {
+        this->source.preRelationId = newAddress;
+
```

Comment:
Remove empty line

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/115/commits/ed31ba0af33e9629173808960eef3cfe90c5dbe9

Reply:
Done